### PR TITLE
Changes

### PR DESCRIPTION
**Pull request by MilenaZurmuehl**
- Fixed issues with SearchShift-functionality
- Started working on SearchProduct/ActivityDiagram-functionality
- Deleted all unnessary files
- renamed 'offerBean'--> new name 'OfferProductBean'
- renamed 'offerDAO'--> new name 'OfferProductDAO'
- renamed 'offerServlet'--> new name 'OfferProductServlet'
- renamed 'offer.jsp'--> new name 'offerProduct.jsp'

### CODE REVIEW

File: BaseProject/WebContent/VisitorSearch.jsp [hunk 1]
```diff
@@ -29,18 +28,10 @@
 	</form>
 </br>
 <% if (request.getAttribute("error") != null) { %>
-
```
Comment (vincentdelitz) [inline BaseProject/WebContent/VisitorSearch.jsp]: Why did you delete that? That is part of my solution. So please don't modify this file.

**Review by vincentdelitz — CHANGES_REQUESTED** [review]
I left a comment for you in VisitorSearch.jsp, because you deleted my functionality. Moreover, if I insert a new product into the database, a blank page appears instead of the actual table.

Does that work for you? If not, please fix it.

**Review by vincentdelitz — APPROVED** [review]

File: BaseProject/WebContent/offerProduct.jsp [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+			</thead>
+			<% ArrayList<OfferProductBean> offers = (ArrayList<OfferProductBean>)session.getAttribute("offers"); %>
+			<tbody> 
+
+				<% for (int i=0; i < offers.size(); i++) { 
+					offer = (OfferProductBean)offers.get(i); %>
+					<tr>
+						<td><a href="./locationServlet?product_id=<%=offers.get(i).getProductID()%>&shop_id=<%=offers.get(i).getShopID()%>">See Location</a></td>
+						<td><%=offers.get(i).getProductID()%></td>
+						<td><%=offers.get(i).getPname()%></td>
```
Comment (demirdagemir) [inline BaseProject/WebContent/offerProduct.jsp:52]: I'm not sure if this is a DB issue but Pname is always `null` in my case.
